tensorflow/lite/kernels/internal/reference/softmax.h

```diff
@@ -145,6 +144,20 @@ inline void Softmax(const SoftmaxParams& params,
   }
 }
 
+int16_t CalculateExp(const SoftmaxParams& params, const int16_t* input_data,
```

Comment:
`inline int16_t CalculateExp`
currently test failed with multiple definition

meanwhile, I think it's ok to remove container and calculate `exp_result_Q015` as needed.

Reply:
Which of the tests failed? Everything passed for me.
If the issue is doing the calculation in CalculateExpI can move it back into the main loop, moved it out for readability.

Reply:
Hi, it appears compile error, when I run test:
`bazel test tensorflow/lite/kernels:activations_test --test_output=all`

Reply:
Ah, I see, multiple defs. I inlined the CalculateExp function and all seem to pass, see latest commit.